test/src/test/java/org/corfudb/runtime/clients/LogUnitHandlerTest.java

```diff
@@ -118,6 +118,16 @@ public void canReadWrite()
                 .isEqualTo(testString);
     }
 
+    @Test
+    public void canGetTrimMarkAndLogSize()
+            throws Exception {
+        byte[] testString = "hello world".getBytes();
+        client.write(0, null, testString, Collections.emptyMap()).get();
+        LogData r = client.read(0).get().getAddresses().get(0L);
+        assertThat (client.getTrimMark().get());
+        assertThat (client.getLogFileSize ().get());
```

Comment:
Can you make stonger assertions here, assertThat (client.getLogFileSize ().get()) is greater than "hello world".getBytes() etc..